backend/frege/repositories/tasks.py

```diff
@@ -84,19 +84,23 @@ def _finalize_repo_analysis(repo_obj):
 
 def _clone_repo(repo: Repository, local_path: Path) -> Optional[git.Repo]:
     _check_download_folder_size()
-    
+
     try:
         repo_obj = git.Repo.clone_from(repo.git_url, local_path)
         repo.fetch_time = timezone.now()
         repo.save(update_fields=["fetch_time"])
         logger.info(f"Repository {repo.git_url} cloned")
         return repo_obj
     except Exception as e:
-        logger.error(f"Unexpected error while processing repository {repo.git_url}: {e}")
+        if isinstance(e, git.exc.GitCommandError):
+            logger.error(f"Git error while cloning repository {repo.git_url}.")
+        else:
+            logger.error(f"Unexpected error while processing repository {repo.git_url}: {e}")
```

Comment:
It was clearer that way.

```suggestion
      except git.exc.GitCommandError as e:
          logger.error(f"Git error while cloning repository {repo.git_url}.")
      except Exception as e:
          logger.error(f"Unexpected error while processing repository {repo.git_url}: {e}")
```